docs/general/installation/linux.md

```diff
@@ -14,20 +69,20 @@ Jellyfin can be found in the `community` repository as
 To enable the web UI after installing `jellyfin-web`, make sure to remove the `--nowebclient` option from
 `/etc/conf.d/jellyfin`.
 
-## Arch Linux
+### Arch Linux
 
 The `Extra` repository contains builds for both [`jellyfin-server`](https://archlinux.org/packages/?name=jellyfin-server) and [`jellyfin-web`](https://archlinux.org/packages/?name=jellyfin-web).
 `jellyfin-server` includes a hard dependency on [`jellyfin-ffmpeg`](https://archlinux.org/packages/?name=jellyfin-ffmpeg).
 
 Both packages, server and web, can also be built from source at the tip of the master branch using [`jellyfin-git`](https://aur.archlinux.org/packages/jellyfin-git/).
 The AUR also offers each separately at [`jellyfin-server-git`](https://aur.archlinux.org/packages/jellyfin-server-git/) and [`jellyfin-web-git`](https://aur.archlinux.org/packages/jellyfin-web-git/).
 
-## Fedora
+### Fedora
 
 Fedora builds in RPM package format are available [in the main download repository](https://repo.jellyfin.org/?path=/server/). We do not yet have an official Fedora repository, but one is planned for the future.  
```

Comment:
Since you're already editing this @felix920506 can you adjust this line too to remove the "official RPM" bits and only mention RPMFusion?